meson.build

```diff
@@ -447,7 +447,7 @@ if get_option('build_backends')
       )
 	 files += cuda_files
     files += cuda_gen.process(cuda_files_nvcc_common)
-    files += cuda_gen.process(cuda_files_nvcc_fp16, extra_args: ['-arch=compute_70', '-code=sm_70'])
+    files += cuda_gen.process(cuda_files_nvcc_fp16, extra_args: ['-arch=compute_75', '-code=sm_80', '-code=sm_75', '-code=sm_86'])
```

Comment:
I don't see `sm_86` in the nvcc [documentation](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html).
It is causing me issues since nvcc doesn't recognize it.

Reply:
sorry about that (that's supposedly the SM version for GA10x/RTX30xx GPUs). I have removed it and replaced it simply with  '-arch=compute_53' as suggested by borg323 which seems to work (tested on Turing and Ampere) without any perf degradation.